clock_test.go

```diff
@@ -6,17 +6,75 @@ import (
 	"time"
 )
 
-func TestItComparesTimesCorrectly(t *testing.T) {
-	startTime := time.Now()
```

Comment:
Seems silly and unlikely, but running this test between 23:59:58 and 00:00:02 could have caused failures